models/actions/artifact.go

```diff
@@ -78,6 +78,12 @@ func CreateArtifact(ctx context.Context, t *ActionTask, artifactName, artifactPa
 	} else if err != nil {
 		return nil, err
 	}
+
+	artifact.ExpiredUnix = timeutil.TimeStamp(time.Now().Unix() + 3600*24*expiredDays)
```

Comment:
![image](https://github.com/user-attachments/assets/77d1dc2b-1f5b-423d-8879-ad034e4677a6)
we have this

Reply:
374d323eaadaf555168fe3c4cb94ac39c2b14245